src/commands/postnsave.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+
+import FeedParser from 'feedparser';
+import request from 'request';
+import fs from 'fs';
+
+import {owner} from '../config/config.json';
+
+import type {Message} from 'discord.js';
+import type {Stash} from '../type/Stash';
+
+let url = 'https://www.craneanime.com/wp/feed/atom';
+// let url = 'https://xkcd.com/atom.xml';
+
+export default {
+  name: 'postnsave',
+  description: 'Check the atom feed',
+  guildOnly: true,
+  execute(message: Message, args: String) {
+    // console.log('message', message.guild.id);
+    let newArticles = [];
+    if (message.author.id !== owner) {
+      return;
+    }
+
+    message.channel.send('Checking feed...').then((sentMessage) => {
+      if (Array.isArray(sentMessage)) {
+        return;
+      }
+
+      sentMessage.delete(3000);
+    });
+
+    checkFeed(url).then(async(result) => {
```

Comment:
You don't need `async` if you don't plan to use `await`.

Reply:
oh yeah, wew bad habit of using but not analysing through 😸